qwp_query_decoder_test.go

```
@@ -0,0 +1,2230 @@
+	if aliased[0].name != "a" {
+		t.Errorf("alias corrupted by clear: name=%q", aliased[0].name)
+	}
+}
+
+func assertDecodeErrContains(t *testing.T, err error, substr string) {
+	t.Helper()
+	if err == nil {
+		t.Fatalf("expected error containing %q, got nil", substr)
+	}
+	var de *qwpDecodeError
+	if !errors.As(err, &de) {
```

golangci-lint (2.11.4)

[error] SA9003: empty branch

(staticcheck)